### DISCUSSION TITLE
Delay reading Serial port

### DISCUSSION THREAD

**Discussion by 1XU7** [videofeedback/Unreal-Engine-Plugin-Communication-Serial-Port#18] [General]
Hi!.. thanks for this plugin and video..

i have downloaded version 5.3.2 and using it with Unreal 5.3.2 as well..
i have wrote a very basic program to control a potentiometer maped from -175 to -20, on the terminal Putty it prints ok and instantly.... but in Unreal, i have a lag, i change the potentiometer and values are printed on the screen some 3 or 4 seconds later... what could be?..

thanks.

**Comment by y45shh** [comment]
I have the same issue!

Ardunio IDE serial monitor displays my potentiometer reading accuratly, but when reading the value in ue5 and printing to the screen, there is a delay too.

**Reply by 1XU7** [reply]
agreed, that confirms my position, thanks!..

**Comment by Fishwil** [comment]
omg no one replies... have you guys solve the problems ? btw mine is not even detected in UE5... so rare to find a solution...

**Comment by y45shh** [comment]
nope sorry :(
